bedrock/firefox/templates/firefox/windows-64-bit.html

```diff
@@ -0,0 +1,76 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% extends "base-protocol.html" %}
+
+ {% add_lang_files "mozorg/technology/windows-64-bit" %}
+
+ {% from "macros-protocol.html" import call_out_compact, hero with context %}
```

Comment:
This isn't breaking anything, but afaik macros should be imported before `{% extends...` is called above.